src/info/info.c

```diff
@@ -165,13 +165,14 @@ typedef struct InfoLoadData
 } InfoLoadData;
 
 static void
-infoLoadCallback(void *data, const String *section, const String *key, const String *value)
+infoLoadCallback(void *data, const String *section, const String *key, const String *value, const Variant *valueVar)
```

Comment:
I don't understand why we need the `String *value` - it is only used in `INFO_CHECKSUM_KEY_VALUE(loadData->checksumActual, key, value);` so why not pass `varStr(valueVar)` instead and remove `String *value` altogether??

Reply:
varStr() won't work because it might not be a string -- also it needs to be a string in JSON format. So, if we don't return the string we need to re-encode it to JSON to calculate the checksum.

Also, if we decide to use this function for loading configs it will need to be able to return a normal string.